README.md

```diff
@@ -2,37 +2,49 @@
 
 The following technologies should have been installed on your machine where you will build, test and deploy your applications.
 
-* NodeJS 12.4.1 or above
-* git tool
+- NodeJS 12.4.1 or above
```

Comment:
Are these formatting changes intentional? Just in the interest of minimizing merge conflicts in the future (when we fetch from upstream) I'd say let's try to minimize the changes we make unless they're absolutely necessary

Reply:
They are not intentional. Prettier back at it. I will try and revert although for that last line I'm not entirely sure what the change even is....